test/integration/goldens/logging/ConfigClient.java

```diff
@@ -395,6 +395,23 @@ public final ListBucketsPagedResponse listBuckets(ListBucketsRequest request) {
    * Lists buckets (Beta).
    *
    * <p>Sample code:
+   *
+   * <pre>{@code
+   * try (ConfigClient configClient = ConfigClient.create()) {
+   *   while (true) {
+   *     ListBucketsResponse response = configClient.listBucketsCallable().call(request);
+   *     for (LogBucket element : response.getResponsesList()) {
+   *       // doThingsWith(element);
+   *     }
+   *     String nextPageToken = response.getNextPageToken();
+   *     if (!String.isNullOrEmpty(nextPageToken)) {
```

Comment:
`Strings.isNullOrEmpty` - this is a Guava method.

Reply:
Match the original sample code. Here is an [example](https://github.com/googleapis/java-logging/blob/release-v1.101.3/google-cloud-logging/src/main/java/com/google/cloud/logging/v2/ConfigClient.java#L294).

Do we need to make any change here?

Reply:
Gotcha. Replace "String" by "Strings".